contracts/feature-tests/composability/forwarder/src/call_sync.rs

```diff
@@ -7,74 +7,6 @@ pub trait ForwarderSyncCallModule {
     #[proxy]
     fn vault_proxy(&self) -> vault::Proxy<Self::Api>;
 
-    #[endpoint]
-    #[payable("*")]
-    fn echo_arguments_sync(
```

Comment:
Please don't delete this, it is still a valid test.

Reply:
restored